core/drivers/bcm_sotp.c

```diff
@@ -111,25 +128,155 @@ TEE_Result bcm_iproc_sotp_mem_read(uint32_t row_addr, uint32_t sotp_add_ecc,
 	/* No ECC check till SOTP_NO_ECC_ROWS */
 	if (row_addr > SOTP_NO_ECC_ROWS &&
 	    reg_val & SOTP_STATUS_1__ECC_DET) {
-		EMSG("SOTP ECC ERROR Detected ROW %d\n", row_addr);
+		EMSG("SOTP ECC ERROR Detected ROW %d", row_addr);
```

Comment:
here also, `row_addr` is `uint32_t` hence:
`EMSG("SOTP ECC ERROR Detected ROW %"PRIu32, row_addr);`